app/helpers/application_helper.rb

```diff
@@ -9,6 +9,6 @@ def license_label(id)
 
   # A work uses its thumbnail for preview; file just uses itself.
   def preview_id(presenter)
-    presenter.respond_to?(:thumbnail_id) ? presenter.thumbnail_id : presenter.id
+    presenter.respond_to?(:thumbnail_id) and presenter.thumbnail_id ? presenter.thumbnail_id : presenter.id
```

Comment:
Oh, I have a couple areas where python style has stuck on me. I appreciate these things being pointed out so I can be more rubyish. I'll change it!